inst/rstudio/addins.dcf

```diff
@@ -3,3 +3,7 @@ Description: Select Crunch Dataset
 Binding: listDatasets(shiny = TRUE)
 Interactive: true
 
+Name: Array Builder
```

Comment:
Should this be in `crunchy` instead of `crunch`?

Reply:
It should be, I thought we'd do this in a couple of PRs to make it a bit easeir to id the PR which brought each gadget into the two packages.